dipy/reconst/sfm.py

```diff
@@ -337,9 +428,9 @@ def predict(self, gtab=None, response=None, S0=None):
             S0 = self.S0
         if isinstance(S0, np.ndarray):
             S0 = S0[..., None]
-        if isinstance(self.mean_signal, np.ndarray):
-            mean_signal = self.mean_signal[..., None]
-        pre_pred_sig = S0 * (pred_weighted + mean_signal)
+        if isinstance(self.iso, np.ndarray):
```

Comment:
What is this meant to do?  Guarantee 2D-ness?  Then rather use an explicit array constructor with `copy=False`.